src/decoration.c

```diff
@@ -494,7 +520,7 @@ static u8 AddDecorationWindow(u8 windowIndex)
     if (windowIndex == 0)
     {
         template = sDecorationWindowTemplates[0];
-        template.width = GetMaxWidthInMenuTable(sDecorationMainMenuActions, ARRAY_COUNT(sDecorationMainMenuActions));
+        template.width = GetMaxWidthInMenuTable(sDecorationMainMenuActions, NELEMS(sDecorationMainMenuActions));
```

Comment:
Please use `ARRAY_COUNT`. It's the more readable of the two macros (and the standard for pokeemerald).